Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferPrivateAVFObjC.mm

```diff
@@ -831,7 +831,7 @@ static bool sampleBufferRenderersSupportKeySession()
         }
     }
 
-    if (auto* player = this->player())
+    if (RefPtr player = this->player())
```

Comment:
I am not sure what our coding style says here or whether we get consensus.
Normally, I would think auto should do the right thing, I would tend to do s/auto*/auto.

Reply:
I discussed this with Chris on slack, and we are moving to using this notation as we are moving to use more smart pointer.